### PR TITLE
AdditionalTags Workload Param  

### PR DESCRIPTION
**Pull request by mgibson1121**
Buildpack upgrade instructions should be included in all supported versions of TAP. 

AdditionalTags param only in 1.7

### CODE REVIEW

File: tanzu-build-service/dependencies.hbs.md [hunk 1]
```diff
@@ -202,5 +202,18 @@ version of Tanzu Application Platform.
...
-For more information about updating a stack and a buildpack, see [Cluster stacks update](https://docs.vmware.com/en/Tanzu-Build-Service/1.12/vmware-tanzu-build-service/updating-deps.html#cluster-stacks-update) and [Cluster store update](https://docs.vmware.com/en/Tanzu-Build-Service/1.12/vmware-tanzu-build-service/updating-deps.html#cluster-store-update) in the Tanzu Build Service documentation. Both workflows require the [kp CLI](https://network.pivotal.io/products/build-service).
+### Upgrading Buildpacks out-of-band of TAP 
+
+Updating buildpack dependencies outside of upgrades to Tanzu Application Platform is possible, but VMware recommends upgrading TAP in order to consume new build dependencies. Should you encounter the need to upgrade buildpacks between TAP releases, follow these instructions:
+
+1. Navigate to the the desired buildpack page. Selecting from the Tanzu Buildpacks documentation <a href="https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-index.html">landing page</a> will redirect to the TanzuNet distribution page for a given language family buildpack.
```
Comment (chenbh) [inline tanzu-build-service/dependencies.hbs.md:209]:
```suggestion
1. Navigate to the the desired buildpack page. Selecting from the Tanzu Buildpacks documentation [landing page](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-index.html) will redirect to the TanzuNet distribution page for a given language family buildpack.
```

File: tanzu-build-service/dependencies.hbs.md [hunk 2]
```diff
@@ -202,5 +202,18 @@ version of Tanzu Application Platform.
...
+2. Find the appropriate package in the artifact reference list. Select `lite` if you have installed TAP using `lite` TBS dependencies. This is the default set of dependencies installed. If you installed `full` dependencies, select the package artifact reference that is **not** identified as lite.
+3. Click the icon in the right side of the artifact reference box to reveal the Tanzu Network image repository address and image tag and copy to clipboard.
+4. Relocate buildpack image using imgpkg copy 
+`imgpkg copy -b <PASTE TANZUNET IMAGE URL FROM PREVIOUS STEP> --to-repo ${INSTALL_REGISTRY_HOSTNAME}/${INSTALL_REPO}/tbs-deps/<BUILDPACK LANGUAGE>`
```
Comment (chenbh) [inline tanzu-build-service/dependencies.hbs.md:213]:
```suggestion
4. Relocate buildpack image using imgpkg copy 
`imgpkg copy -b <PASTE TANZUNET IMAGE URL FROM PREVIOUS STEP> --to-repo ${INSTALL_REGISTRY_HOSTNAME}/${INSTALL_REPO}/tbs-deps/<BUILDPACK LANGUAGE>`
  Where `$INSTALL_REGISTRY_HOSTNAME` and `$INSTALL_REPO` are the same ones from the [online profile installation docs](../install-online/profile.md#relocate-images)
```

**Comment by kcoriordan** [comment]
If Buildpacks are packaged separately from TBS, it might be good to have the install/upgrade doc separated from TBS also and added under the new Buildpacks section.

**Comment by mgibson1121** [comment]
@kcoriordan This should be good to merge. The buildpack upgrade instructions should be superseded by https://github.com/pivotal/docs-tap/pull/3083

**Pull request merged**